src/UCommerce.SiteCore.Installer/Steps/RemoveRenamedPipelines.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.IO;
+using Ucommerce.Installer;
+
+namespace Ucommerce.Sitecore.Installer.Steps
+{
+    public class RemoveRenamedPipelines : AggregateStep
+    {
+        private readonly IInstallerLoggingService _loggingService;
+
+        public RemoveRenamedPipelines(DirectoryInfo sitecoreDirectory, IInstallerLoggingService loggingService)
+        {
+            _loggingService = loggingService;
+            var pipelinesPath = Path.Combine("sitecore modules", "Shell", "ucommerce", "Pipelines");
```

Comment:
Couldn't you create a pipelinesDirectory DirectoryInfo object here so you can do something like the below when you you create the DeleteFileWithBackup objects?

```c#
new DeleteFileWithBackup(new FileInfo(Path.Combine(pipelinesDirectory.FullName, "Basket.config")), _loggingService)
```